meson.build

```diff
@@ -1,7 +1,7 @@
 project(
     'pyvrp',
     'cpp',
-    version: run_command('poetry', 'version', '--short', check: true).stdout(),
+    # version: run_command('poetry', 'version', '--short', check: true).stdout(),
```

Comment:
Can be called with `uv version` from uv 0.7.0 onwards.

https://github.com/astral-sh/uv/releases/tag/0.7.0